test/blocks/global-footer/global-footer.test.js

```diff
@@ -272,4 +272,78 @@ describe('global footer', () => {
       }
     });
   });
+
+  describe('LANA logging tests', () => {
+    const originalLanaLog = window.lana.log;
+    let lanaLogSpy;
+
+    beforeEach(async () => {
+      lanaLogSpy = sinon.spy();
+      window.lana.log = lanaLogSpy;
+    });
+
+    afterEach(() => {
+      window.lana.log = originalLanaLog;
```

Comment:
Could we not stub and reset, instead of saving the original implementation?

Reply:
Done